uber/models/department.py

```diff
@@ -407,6 +416,38 @@ def hours(self):
     def end_time(self):
         return self.start_time + timedelta(hours=self.duration)
 
+    def working_limit_ok(self, attendee):
+        attendee_hour_map = attendee.hour_map
+        hours_worked = self.duration
+        working_hour_limit = self.department_max_consecutive_hours
+        if working_hour_limit == 0:
+            working_hour_limit = 1000 # just default to something large
+
+        # count the number of filled hours before this shift
+        cur = self.start_time - timedelta(hours=1)
```

Comment:
For readability, can we change this to something like `current_shift_hour` or `current_shift` (or `cur_shift` is fine too)? I had to read through the full block before I understand what thing "current" was referring to.